AnnoyancesFilter/sections/popups.txt

```diff
@@ -2590,6 +2591,7 @@ saocarlosagora.com.br#$?#html.fancybox-lock:has(> body) { overflow: visible!impo
 m.hh.ru##div[data-qa$="-banner"]
 !+ PLATFORM(android, ios, ext_android_cb)
 maxfoodfun.com##.pixnet-happix-pilot__launcher-bar__gift-notification
+indianexpress.com###append_cubebox_box
```

Comment:
As previous. Better hide `#append_breaking_box`

Reply:
Thats actually the element that was reported in the issue, floating animated popup

Reply:
Not all reported content must be blocked.